code/_core/datum/reagent/reagent_magic.dm

```diff
@@ -24,4 +24,59 @@
 		L.add_stand()
 		if(current_volume) container.remove_reagent(src.type,current_volume,should_update = FALSE, check_recipes = FALSE)
 		return 0
+*/
+
+/reagent/magic/armotol
```

Comment:
This should be a subtype of /reagent/drug/ as it already has a thing to handle status effects. Check out the file reagent_drug.dm for how it works.

Reply:
Completed, thanks for showing me that.